ten-simple-rules-dockerfiles.Rmd

```diff
@@ -524,6 +556,9 @@ This variety of approaches render seemingly more convenient uncontainerised envi
 # 8. Establish templates for new projects {-}
 \rulelabel{8}{rule:templates}
 
+
+[SJE: this seemed a bit lacking in meat, compared to the other rules.  Do you have an example Dockerfile template that you are thinking of?]
+
```

Comment:
This rule says to me essentially what we've said in another rule - "Don't start with a blank file, copy paste something that is almost what you want." I would nix this section and replace with something about automation to clean up / shorten the section that is too long.